glob_test.go

```diff
@@ -239,44 +258,44 @@ func TestGlob(t *testing.T) { // nolint:funlen
 
 	t.Run("match files in directories", func(t *testing.T) {
 		t.Parallel()
-		matches, err := Glob("/a/{b,c}", WithFs(testFs(t, []string{
-			"/a/b/d",
-			"/a/b/e/f",
-			"/a/c",
+		matches, err := Glob("a/{b,c}", WithFs(testFs(t, []string{
```

Comment:
API-wise it should still be possible to do these tests with a leading `/`, right?

Reply:
never was actually, test was passing but shouldn't...

Reply:
Wow...
Well I would think that it should be possible to glob with absolute paths, though...